VideoWeb/VideoWeb/ClientApp/src/app/services/participant-status-update.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { VideoWebService } from 'src/app/services/api/video-web.service';
+import { EventType, UpdateParticipantStatusEventRequest } from 'src/app/services/clients/api-client';
+import { Logger } from 'src/app/services/logging/logger-base';
+import { Router } from '@angular/router';
+import { pageUrls } from 'src/app/shared/page-url.constants';
+
+export const ParticipantPages: string[] = [
+    pageUrls.Introduction,
+    pageUrls.EquipmentCheck,
+    pageUrls.SwitchOnCameraMicrophone,
+    pageUrls.ParticipantSelfTestVideo,
+    pageUrls.CameraWorking,
+    pageUrls.MicrophoneWorking,
+    pageUrls.VideoWorking,
+    pageUrls.HearingRules,
+    pageUrls.Declaration
+];
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ParticipantStatusUpdateService {
```

Comment:
Wrapping a service to call another service adds a layer of complexity we do not need.

1. Either remove this service and call videoWebService 
or 
2. Call the apiClient directly in this service

Reply:
done